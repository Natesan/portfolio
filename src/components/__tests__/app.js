import React from "react"
import renderer from "react-test-renderer"

import App from "../app"

describe("App", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <App>
          <div id="test"></div>
        </App>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
