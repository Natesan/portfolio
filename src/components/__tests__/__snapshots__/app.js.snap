// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<div
  className="app"
>
  <div
    id="test"
  />
</div>
`;
